.golangci.yml

```diff
@@ -4,8 +4,8 @@ run:
     - apparmor
     - seccomp
     - selinux
-  concurrency: 6
-  deadline: 5m
+  # Don't exceede number of threads available when running under CI
```

Comment:
nit of a nit, only worry about if other stuff is found.
exceede -> exceed

Reply:
Woops, okay.  Will fix this along with anything @edsantiago comes up with (he always finds something) :+1:

Reply:
okay fixed.